demo/tests/test_defaults_with_accesses.py

```diff
@@ -0,0 +1,89 @@
+from django.test import TestCase
+from formidable.forms import (
+    FormidableForm, fields
+)
+from formidable import constants
+from formidable.forms import get_dynamic_form_class
+
+
+class AccessesTestCaseTestForm(FormidableForm):
+    editable_text = fields.CharField(
+        label='Editable text',
+        accesses={'padawan': constants.EDITABLE,
+                  'jedi': constants.EDITABLE,
+                  'human': constants.EDITABLE,
+                  'robot': constants.EDITABLE},
+        default='Default value (editable)'
+    )
+    readonly_text = fields.CharField(
+        label='Readonly text',
+        accesses={'padawan': constants.READONLY,
+                  'jedi': constants.READONLY,
+                  'human': constants.READONLY,
+                  'robot': constants.READONLY},
+        default='Default value (readonly)'
+    )
+    hidden_field = fields.CharField(
+        label='Hidden text',
+        accesses={'padawan': constants.HIDDEN,
+                  'jedi': constants.HIDDEN,
+                  'human': constants.HIDDEN,
+                  'robot': constants.HIDDEN},
+        default='Default value (hidden)'
+    )
+
+
+class AccessesTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        form_class = AccessesTestCaseTestForm
+        self.formidable = form_class.to_formidable(label='title')
+
+    def get_form_class(self, formidable, role):
```

Comment:
not sure of the relevance of this method. Within each call, the formidable is always `self.formidable` and the role is always `jedi`.
I guess that each call can be replaced with:

```
form_class = get_dynamic_form_class(self.formidable, 'jedi')
```

of if you think this method is still relevant, you may want to to remove its arguments or make them optional:

```python
def get_form_class(self, role='jedi', formidable=None):
    if not formidable:
        formidable = self.formidable
    return get_dynamic_form_class(formidable, role)
```

Reply:
You're right I'll remove this method!